pkgs/top-level/all-packages.nix

```diff
@@ -991,6 +991,8 @@ with pkgs;
 
   amidst = callPackage ../tools/games/minecraft/amidst { };
 
+  bikeshed = python3Packages.callPackage ../applications/misc/bikeshed { };
```

Comment:
Don't do this. Put `python3` as parameter instead and use the normal `callPackage`

Reply:
Hmm, I was just following the other python-based targets, like `bashate` and friends.
Do you want me to add `python3Packages` to the top of this file and fix all occurrences of it used inside?
Wouldn't `callPackage` from `Python3Packages` conflict with the `callPackage` used for all other things here?
This seems like a problem that should be solved separately, once and for all, not as a part of this PR.